docs/tsa-policy.md

```diff
@@ -421,17 +395,6 @@ In particular:
   external organizations supporting the TSA services including
   the applicable policies and practices.
 
-- The TSA shall make available to subscribers and relying parties its
```

Comment:
I think this and NOTE 2 is fine to leave - "Make available documentation, as necessary", and the note says "we might not". You could change "shall" to "may" if you want to make it clearer that it's optional.

For example, if the TSA were to log timestamps to a transparency log, that should be publicly documented for auditability.

Reply:
Changing it to *may* is fine!